File: project/ponto/source/sistemas/admin/models/Perfil.php
<?php
/**
 * Modelo da tabela perfil
 *
 * @name Perfil
 * @author Alexis Moura
 * @package _models
 * @version 1.0
 */
class Perfil extends Axs_Db_Table
{
	/**
	 * Array de dados do modelo
	 *
	 * @name $_data
	 * @access public
	 * @var array
	 */
	public $_data = array(
		'idSexo' => null,
		'idOlho' => null,
		'idCabelo' => null,
		'idEtinia' => null,
		'idEstadoCivil' => null,
		'idEndereco' => null,
		'idPerfil' => null,
		);
	/**
	 * Array de validação do modelo
	 *
	 * @name $_validators
	 * @access protected
	 * @var array
	 */
	protected $_validators = array(
		'idSexo' => array( 'notEmpty','presence'=>'required'),
		'idOlho' => array( 'allowEmpty'=>'true'),
		'idCabelo' => array( 'allowEmpty'=>'true'),
		'idEtinia' => array( 'allowEmpty'=>'true'),
		'idEstadoCivil' => array( 'allowEmpty'=>'true'),
		'idEndereco' => array( 'notEmpty','presence'=>'required'),
		'idPerfil' => array( 'allowEmpty'=>'true'),
		);
	/**
	 * Array de descrição dos dados
	 *
	 * @name $descriptionFields
	 * @access public
	 * @var array
	 */
	public $descriptionFields = array(
		'idSexo' => 'Sexo',
		'idOlho' => 'Olho',
		'idCabelo' => 'Cabelo',
		'idEtinia' => 'Etinia',
		'idEstadoCivil' => 'Estado civil',
		'idEndereco' => 'Endereço',
		'idPerfil' => 'Código',
		);
	

}

?>

File: project/CSharp/ponto/PontoEncontro.Domain/Entity/Perfil.cs
//
//  Copyright (c) 2012, AXS 
//  All rights reserved.
//
//  Authors: 
//          
//           * Alexis Moura
//           Generation
//           Messenger:
//
using System;  
using System.Collections.Generic;
 
namespace PontoEncontro.Domain 
{
    /// <summary>
    /// Classe Modelo de Perfil
    /// </summary>
    public class Perfil 
    {  
        #region properties
    
        
            /// <summary>
            ///  idSexo 
            /// </summary>
            public virtual int idSexo { get; set; }

            /// <summary>
            ///  idOlho 
            /// </summary>
            public virtual int idOlho { get; set; }

            /// <summary>
            ///  idCabelo 
            /// </summary>
            public virtual int idCabelo { get; set; }

            /// <summary>
            ///  idEtinia 
            /// </summary>
            public virtual int idEtinia { get; set; }

            /// <summary>
            ///  idEstadoCivil 
            /// </summary>
            public virtual int idEstadoCivil { get; set; }

            /// <summary>
            ///  idEndereco 
            /// </summary>
            public virtual int idEndereco { get; set; }

            /// <summary>
            ///  idPerfil 
            /// </summary>
            public virtual int idPerfil { get; set; }

            /// <summary>
            ///  Sexo
            /// </summary>
            public virtual Sexo sexo { get; set; }

            /// <summary>
            ///  Olho
            /// </summary>
            public virtual Olho olho { get; set; }

            /// <summary>
            ///  Cabelo
            /// </summary>
            public virtual Cabelo cabelo { get; set; }

            /// <summary>
            ///  Etinia
            /// </summary>
            public virtual Etinia etinia { get; set; }
            
            /// <summary>
            ///  Estado civil
            /// </summary>
            public virtual EstadoCivil estadoCivil { get; set; }

            /// <summary>
            ///  Endereço
            /// </summary>
            public virtual Endereco endereco { get; set; }
                       

        #endregion
        
        #region constructors

            public Perfil()
            {
            }
   
        #endregion
    }
}
